### PR TITLE
HDDS-4080. Create ZH translation of OzoneManager.md in doc.

### PR DESCRIPTION
**Pull request by yuyang733**
## What changes were proposed in this pull request?

Create ZH translation of OzoneManager.md in doc.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4080

### CODE REVIEW

**Review by runzhiwang — APPROVED** [review]
LGTM

File: hadoop-hdds/docs/content/concept/OzoneManager.zh.md [hunk 1]
```diff
@@ -64,10 +64,60 @@ OM 使用 Apache Ratis（Raft 协议的一种实现）来复制 OM 的状态，
...
+ * 与FS相关的调用 (针对分层查询而非对象存储的扁平化命名空间进行了优化)
+   * GetFileStatus, CreateDirectory, CreateFile, LookupFile
+ * ACL 相关
+   * 使用 [内部 ACLs]({{< ref "security/SecurityAcls.md" >}}) 代替 [Ranger]({{< ref "security/SecurityWithRanger.md" >}}) 管理访问控制列表
+ * Delegation token (获取 / 刷新 / 取消)
+   * 用于安全相关
+ * 管理API
+   * 获取S3的访问密钥
+   * ServiceList (用于服务发现)
+   * DBUpdates (用于 [Recon]({{< ref "feature/Recon.md" >}}) 下载快照)
```
Comment (cku328) [inline hadoop-hdds/docs/content/concept/OzoneManager.zh.md:94]:
```suggestion
   * DBUpdates (用于 [Recon]({{< ref path="feature/Recon.md" lang="en" >}}) 下载快照)
```
To reference documents for different languages, you need to configure the "lang" parameter.

**Review by cku328 — CHANGES_REQUESTED** [review]
Thanks @yuyang733 for working on this.

I think we should ensure that some of the words are consistent with other existing documents.
`Container` -> `容器`,
`Volume` -> `卷`,
`Bucket` -> `桶`
`Key` -> `键`,etc.

And some suggestions on formatting.

Insert a space around an English word.
`了解Ozone Manager` -> `了解 Ozone Manager`
Parentheses should use full width characters.
(Both formats exist in existing documents, should they be unified?)
`分块上传 (Initiate, Complete…)` -> `分块上传（Initiate, Complete…）`

File: hadoop-hdds/docs/content/concept/OzoneManager.zh.md (hunk 1 above)
Comment (cku328) [reply hadoop-hdds/docs/content/concept/OzoneManager.zh.md:94]: image

**Comment by yuyang733** [comment]
> Thanks @yuyang733 for working on this.
> 
> I think we should ensure that some of the words are consistent with other existing documents.
> `Container` -> `容器`,
> `Volume` -> `卷`,
> `Bucket` -> `桶`
> `Key` -> `键`,etc.
> 
> And some suggestions on formatting.
> 
> Insert a space around an English word.
> `了解Ozone Manager` -> `了解 Ozone Manager`
> Parentheses should use full width characters.
> (Both formats exist in existing documents, should they be unified?)
> `分块上传 (Initiate, Complete…)` -> `分块上传（Initiate, Complete…）`

done.

**Review by cku328 — CHANGES_REQUESTED** [review]
Thanks @yuyang733 for working on this.

Please add some images that are missing (**OM Overview, Key Write and Key Reads**).
I have some suggested changes. We can discuss it together.

**Comment by yuyang733** [comment]
@cku328 Thanks for your comments and all are fixed. Can you help me to review it again?

**Review by cku328 — APPROVED** [review]
LGTM+1.
Thanks @yuyang733 for updating the patch.

**Pull request merged**